components/compliance-service/api/profiles/server/migrate.go

```diff
@@ -67,8 +67,7 @@ func (srv *PGProfileServer) migrateDiskProfiles() error {
 
 			_, err = srv.storeProfile(namespace, tmpFile)
 			if err != nil {
-				logrus.WithError(err).Errorf("Could not store profile %q", tmpFile)
```

Comment:
This error message included the file name of a temp file created during migration rather than the actual file name of the profile, which did not seem very useful. In this context we know the name of the profile, so log that instead.